### PR TITLE
[SP-6442] - S1 steps

### PR DESCRIPTION
**Pull request by hagaishapira**

### CODE REVIEW

File: steps/sentinelone/agent/disconnect_from_network/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,15 @@
+# 1. Builder
```
Comment (kostyay) [inline steps/sentinelone/agent/disconnect_from_network/Dockerfile:1]:
```
FROM us-docker.pkg.dev/stackpulse/public/http/request:1.0.0

ARG STEP_BASEPATH
ARG CONFIG_NAME=config.yaml
COPY $STEP_BASEPATH/${CONFIG_NAME} ./env.yaml
```

Comment (hagaishapira) [reply steps/sentinelone/agent/disconnect_from_network/Dockerfile]: 👍

File: steps/sentinelone/events/get/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM us-docker.pkg.dev/stackpulse/public/http/request:1.0.0
+
+ARG STEP_BASEPATH
+ARG CONFIG_NAME=config.yaml
+COPY $STEP_BASEPATH/${CONFIG_NAME} ./env.yaml
```
Comment (InbarPerry) [inline steps/sentinelone/events/get/Dockerfile:5]:
```suggestion
COPY ${STEP_BASEPATH}/${CONFIG_NAME} ./env.yaml
```

for consistency

**Review by InbarPerry — APPROVED** [review]

File: steps/sentinelone/events/get/Dockerfile (hunk 2 above)
Comment (hagaishapira) [reply steps/sentinelone/events/get/Dockerfile]: 👍

**Pull request merged**